src/Symfony/Component/HttpClient/EventSource/MessageEvent.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Soyuka\EventSource;
+
+class MessageEvent
+{
+    public $data;
+    public $lastEventId;
+    public $eventName;
+    public $origin;
+
+    public function __construct($data = '', $lastEventId = null, $eventName = '', $origin = null)
+    {
+        $this->data = $data;
+        $this->lastEventId = $lastEventId;
+        $this->eventName = $eventName;
+        $this->origin = $origin;
+    }
+
+    public static function parse($data): ?self
+    {
+        $list = preg_split('/\r\n|\r|\n/', $data);
+        $lastEventId = $eventName = $data = null;
+
+        foreach ($list as $listItem) {
+            $parts = preg_split('/:[\s]?/', $listItem, 2);
+            $key = $parts[0];
+            $value = ltrim($parts[1] ?? '');
```

Comment:
```suggestion
            $value = $parts[1] ?? '';
```

Reply:
Collect the characters on the line after the first U+003A COLON character (':'), and let value be that string. **If value starts with a single U+0020 SPACE character, remove it from value.**